License Export (#2900)
modelica/ModelicaSpecification: chapters/functions.tex

  {\lstinline!annotation(SourceDirectory="modelica:/ModelicaLibraryName/Resources/Source")!}\annotationindex{SourceDirectory}, gives the location for source files.
  The preceding one is the default and need not be specified; but another location could be specified by using an URI name for the source directory, see \cref{external-resources}.
  It is not specified how they are built.
\item
  The {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives the license file for the function.

>   The license files are human-readable files containing the license conditions.

Author: Rely even more on `\cref{license-texts}`:
```suggestion
```

>   It is analogous to the \lstinline!License!-annotation for a top-level class, see \cref{license-texts}.

Author: I realized earlier today that annotation-names are not hyphened:
```suggestion
  It is analogous to the \lstinline!License! annotation for a top-level class, see \cref{license-texts}.
```

  {\lstinline!annotation(SourceDirectory="modelica:/ModelicaLibraryName/Resources/Source")!}\annotationindex{SourceDirectory}, gives the location for source files.
  The preceding one is the default and need not be specified; but another location could be specified by using an URI name for the source directory, see \cref{external-resources}.
  It is not specified how they are built.
\item

>   The {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives the license files for the function.
>   The license files are human-readable files containing the license conditions.

Author: The annotation syntax only allows specifying a single file, which should be matched by the text:
```suggestion
  The {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives a license file for the function.
  License files are human-readable files containing the license conditions.
```

Author: This begs the question: Can a function have multiple `License`-annotations?

Author: I'll answer my own question by noting that it would probably be better to just refer to how the annotation's definition for a top-level class than to partially repeat the definition here:
```suggestion
  The {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives a license files for the function.
  It is analogous to the \lstinline!License!-annotation for a top-level class, see \cref{license-texts}.
```

Reviewer: The last line is still useful.

Reviewer: Seems resolved.

>   Top-level classes may also contain this annotation (with similar contents), see \cref{license-texts}.
>   When generating a distributable version for a translated model the license files for used functions and top-level classes should be included.

Author: Rely on `\cref{license-texts}` and avoid repetition:
```suggestion
```

Reviewer: This has been removed and the proposed line above added.
